pkg/relayer/proxy/proxy.go

```diff
@@ -65,6 +65,10 @@ type relayerProxy struct {
 	// servedRelaysPublishCh is a channel that emits the relays that have been served so that the
 	// servedRelays observable can fan out the notifications to its subscribers.
 	servedRelaysPublishCh chan<- *types.Relay
+
+	// pingEnabled indicates whether the relay servers should be pinged before starting them.
```

Comment:
I feel like this needs to be rephrased a bit. Maybe something like:

```suggestion
	// pingEnabled indicates whether the relay servers should require a successful ping during startup.
```